File: Task1/task1.php
<?php
$mystring = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
while (true) {
    $number = readline("Введите число: ");
    if (!is_numeric($number) || $number <= 0 || $number > 26) {
        echo "Неправильное число. Попробуйте снова.\n";
        continue;
    }
    $point = $number - 1;
    echo "Результат: \n";
    while ($point >= 0) {
        for ($i = $number - 1; $i >= $point; $i--) {
            echo $mystring[$i];
            echo " ";
        }
        $point -= 1;
        echo "\n";
    }
    break; 
}
?>

File: Task1/task1.cs
using System;
namespace Laba2
{
    class Programm
    {
        static void Main()
        {
            string myString = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
            int number = 0;
            while (true)
            {
                Console.Write("Input number: ");
                string input = Console.ReadLine();
                if (int.TryParse(input, out number) && number > 0 && number <= 26)
                {
                    break; 
                }
                else
                {
                    Console.WriteLine("Wrong number. ");
                }
            }
            Console.WriteLine("Result: ");
            int point = number - 1;
            while (point >= 0)
            {
                for (int i = number - 1; i >= point; i--)
                {
                    Console.Write(myString[i]);
                    Console.Write(" ");
                }
                point--;
                Console.WriteLine();
            }
        }
    }
}
